cdap-data-fabric/src/main/java/co/cask/cdap/data2/transaction/stream/hbase/HBaseStreamConsumerStateStoreFactory.java

```diff
@@ -52,25 +51,24 @@
   @Override
   public synchronized StreamConsumerStateStore create(StreamConfig streamConfig) throws IOException {
     Id.Namespace namespace = streamConfig.getStreamId().getNamespace();
-    byte[] tableName = Bytes.toBytes(getTableName(namespace));
-
+    TableId streamStateStoreTableId = TableId.from(getTableName(namespace));
```

Comment:
how come we're generating a table name from a namespace and then parsing it back?  If we know what the namespace is and know what the table name is, shouldn't we just create the TableId directly?